### PR TITLE
Move sample web app to test-dapp to GitHub pages

### PR DESCRIPTION
**Pull request by Hugoo**
Ticket: <https://app.clickup.com/t/2tt91az>

- Get rid of k8s
- Use gh pages instead

### CODE REVIEW

**Review by dzbo — COMMENTED** [review]
The nice thing about previous workflow was that it split the elements and created dependency. This way you could start workflow from failed step without running whole. 

CleanShot 2022-08-16 at 20 11 54@2x

You also miss caching.

**Comment by Hugoo** [comment]
Indeed, here are my thoughts:

1. A 35 lines CI file is easier to read / understand / adjust over a 168 lines file.
2. The multi jobs CI takes approx 3min to run vs 2min (50% increase) for the current 1 job CI (however, there is only one run so it might take more time sometimes). Spinning a new job adds complexity as the cache needs to be transferred between runners, etc. For the lint steps, the lint action takes <5sec and the "Set up job" takes 4sec. Spinning the new runner is almost as long as running the lint.
3. The content of the steps are fairly easy and super fast, under 45sec, just CSS lint, format, etc. (which could also be done in 1 lint step) - Therefore, splitting all these steps into multiple jobs doesn't add much value and we don't benefit that much of the parallelisation from the jobs imo.

To summarise: I think this CI is too small and too easy to justify the split. I'd rather go for a simple CI rather than a complex one for almost same, if not worse, performances.

**Comment by dzbo** [comment]
> Indeed, here are my thoughts:
> 
> 1. A 35 lines CI file is easier to read / understand / adjust over a 168 lines file.
> 2. The multi jobs CI takes approx 3min to run vs 2min (50% increase) for the current 1 job CI (however, there is only one run so it might take more time sometimes). Spinning a new job adds complexity as the cache needs to be transferred between runners, etc. For the lint steps, the lint action takes <5sec and the "Set up job" takes 4sec. Spinning the new runner is almost as long as running the lint.
> 3. The content of the steps are fairly easy and super fast, under 45sec, just CSS lint, format, etc. (which could also be done in 1 lint step) - Therefore, splitting all these steps into multiple jobs doesn't add much value and we don't benefit that much of the parallelisation from the jobs imo.
> 
> To summarise: I think this CI is too small and too easy to justify the split. I'd rather go for a simple CI rather than a complex one for almost same, if not worse, performances.

Makes sense, let's go for cleaner/quicker solution.

**Review by dzbo — APPROVED** [review]
